app/jobs/bulk_restricted_device_password_emailing_job.rb

```diff
@@ -0,0 +1,47 @@
+class BulkRestrictedDevicePasswordEmailingJob < ApplicationJob
+  queue_as :default
+
+  ALL_SETTINGS = :all
+
+  def perform(rb_limit: ALL_SETTINGS, school_limit: ALL_SETTINGS)
+    @rb_limit = rb_limit
```

Comment:
I could understand this, but I had to read ahead foirst. Maybe `rb_limit_scope` might explain more here about the intention?

Reply:
Good, I will rename it